examples/dt/uni04delta/values-wo-ceph-client.yaml

```diff
@@ -0,0 +1,172 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
```

Comment:
this file seems not used

Reply:
It is used. It gets modified by the ceph-client and the result gets written in the `values.yaml` file in the same directory. 
It is controlled by these variables in the scenario file:
```
_scenario_src_path: "{{ _arch_repo }}/examples/dt/{{ cifmw_architecture_scenario }}"
cifmw_ceph_client_values_post_ceph_path_src: "{{ _scenario_src_path }}/values-wo-ceph-client.yaml"
cifmw_ceph_client_values_post_ceph_path_dst: "{{ _scenario_src_path }}/values.yaml"
cifmw_ceph_client_service_values_post_ceph_path_src: "{{ _scenario_src_path }}/control-plane/service-values-wo-ceph-client.yaml"
cifmw_ceph_client_service_values_post_ceph_path_dst: "{{ _scenario_src_path }}/control-plane/service-values.yaml"
```

Reply:
why not using _src and _dst as same path? as with that it would be usable without relying on ci-framework generation logic.